helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -82,7 +82,12 @@ internal class SplitCommand : CliktCommand(
                 val text = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(listOf(curation))
 
                 outputFile.parentFile.safeMkdirs()
-                outputFile.writeText(text)
+
+                if (outputFile.exists()) {
```

Comment:
Even if not strictly required here, please use `isFile` instead of `exists()`  which is good style to also verify the "thing" is of the expected type.